presto-druid/src/test/java/io/prestosql/plugin/druid/DruidQueryRunner.java

```diff
@@ -39,10 +42,18 @@ private DruidQueryRunner() {}
 
     public static QueryRunner createDruidQueryRunnerTpch(TestingDruidServer testingDruidServer)
             throws Exception
+    {
+        return createDruidQueryRunnerTpch(testingDruidServer, ImmutableMap.of());
```

Comment:
The existing usage is only two places. Could you move `ImmutableMap.of()` to TestDruidIntegrationSmokeTest & TestDruidIntegrationSmokeTestLatest instead of adding a method?

Reply:
done